Evaluation.html

```diff
@@ -5,23 +5,22 @@
 </head>
 <body>
 	<header>
-		<ul class="pageHeader">
-			<li >
+		<ul class="header headerT">
```

Comment:
I'll admit that I'm more familiar with the type of separation that Gabi is suggesting from my time as a Flex developer than after moving to the JS / HTML / CSS front, but from what I can tell, what you did thus far looks good. I'd defer to Gabi for a final answer though, since I've yet to sync up with him regarding his expectations on this.